lib/data/solr_fields_config.yml

```diff
@@ -404,6 +404,10 @@ series_issn:
   type: num
 series_other_ids:
   type: e
+sersol_number_a:
```

Comment:
Field name is just `sersol_number` without `_a`

Reply:
Fixed.